src/xtgeo/grid3d/_roff_grid.py

```diff
@@ -151,7 +157,7 @@ def _create_lookup(self):
                 else:
                     self._lookup[i + 1] = 1 + self._lookup[i]
 
-    def z_value(self, node):
+    def z_value(self, node: tuple[int, int, int]) -> np.ndarray:
```

Comment:
We're using capital `Tuple` everywhere else, should probably be consistent with it